test/bigtest/network/config.js

```diff
@@ -378,10 +387,8 @@ export default function configure() {
     totalRecords: 0
   });
 
-  this.get('/material-types', {
-    mtypes: [],
-    totalRecords: 0
-  });
+  this.get('/material-types', ({ materialTypes }) => materialTypes.all());
```

Comment:
I know that BigTest has a shortcut for that use case, but I know that it may now work every.
```suggestion
  this.get('/material-types');
```

Reply:
Oh I didn't know about that. That's a pretty cool shortcut.